### DISCUSSION TITLE
Which distros are you currently targeting

### DISCUSSION THREAD

**Discussion by andy5995** [AppImage/AppImageKit#1254] [Q&A] [answered]
Is this up to date as far as which distros you're still targeting? I was trying to decide to switch from Bionic to Focal for [the 0ad AppImage build](https://github.com/0ad-matters/0ad-appimage/) when bionic reaches ESS (EOL is in 2028), but now I think the devs here would recommend I try the build on Xenial (I use LinuxDeploy for the build).

https://github.com/AppImage/AppImageKit/blob/5735cc5bed206497cddfbd2a75e1982c2606c35d/README.md?plain=1#L194

**Comment by TheAssassin** [comment, marked as answer]
https://docs.appimage.org/reference/best-practices.html?highlight=lts#binaries-compiled-on-old-enough-base-system could use an update. Rule of thumb: oldest still-supported LTS. That would be bionic at the moment. However, given bionic is going to be EOL in less than a month (!), feel free to go for focal already.

**Reply by probonopd** [reply]
Yes, rule of thumb for making AppImages: the ingredients that go into AppImages should be built on nothing newer than the oldest still-supported Ubuntu LTS, so currently (still) bionic. I never quite understand why application developers have this urge to go to newer distribution versions before the end of the support for the current version (currently: bionic).

The expectation is that AppImages that can run on the oldest still-supported Ubuntu LTS should be able to run on newer distributions as well. (The reverse is not true, unless you bundle _all_ depenencies inside the AppImage and use a static runtime.)

For building the AppImage tools themselves (such as AppImageKit and especially the AppImage runtime that gets embedded into each AppImage), we want to be _way_ more conservative. We try to build AppImageKit on the oldest distributions we can. Developers wishing to make AppImages should normally not need to build AppImageKit themselves.
